index.html

```diff
@@ -0,0 +1,1224 @@
+
+                      <p id="req-server-origin-authorization">When an HTTP request includes the <code>Origin</code> header, the requested operation is granted on the target resource when there is a match for:</p>
+
+                      <ul>
+                        <li>an Authorization allowing access to the requesting agent (<code>acl:agent</code>, <code>acl:agentGroup</code>, <code>acl:agentClass</code>);</li>
+                        <li>an Authorization with an <code>acl:origin</code> property value that of <code>Origin</code>’s field-value, when the required access mode is not available to all agents (<code>acl:agentClass foaf:Agent</code>); and</li>
+                        <li>the required access mode is allowed for the requesting agent and the origin.</li>
+                      </ul>
+
+                      <p id="req-server-cors-acao-acah">When a server participates in the <abbr title="Cross-Origin Resource Sharing">CORS</abbr> protocol [<cite><a class="bibref" href="#bib-fetch">FETCH</a></cite>] and authorization is granted to an HTTP request including the <code>Origin</code> header, the server MUST include the HTTP <code>Access-Control-Allow-Origin</code> and <code>Access-Control-Allow-Headers</code> headers in the response of the HTTP request.</p>
+
+                      <div class="note" id="access-subject-origin-rejection-reason" inlist="" rel="schema:hasPart" resource="#access-subject-origin-rejection-reason">
+                        <h5 property="schema:name"><span>Note</span>: Access Subject and Origin Rejection Reason</h5>
+                        <div datatype="rdf:HTML" property="schema:description">
+                          <p>Implementations are encouraged to describe reasons to reject the request in the HTTP response the difference between the access subject not being allowed and the origin associated with the HTTP request not being granted access.</p>
+                        </div>
+                      </div>
+
+                      <div class="note" id="trusted-origins" inlist="" rel="schema:hasPart" resource="#trusted-origins">
+                        <h5 property="schema:name"><span>Note</span>: Trusted Origins</h5>
+                        <div datatype="rdf:HTML" property="schema:description">
+                          <p>Implementations might implicitly allow a list of origins, such as the same-origin [<cite><a class="bibref" href="#bib-rfc6454">RFC6454</a></cite>].</p>
+                        </div>
+                      </div>
+
+                      <p id="req-server-wac-allow">Servers MUST advertise client’s access privileges on a resource by including the <code>WAC-Allow</code> HTTP header (<cite><a href="#wac-allow" rel="rdfs:seeAlso">WAC-Allow</a></cite>) in the response of HTTP <code>GET</code> and <code>HEAD</code> requests.</p>
```

Comment:
I am unsure why and find it a bit confusing that the WAC-Allow requirement is in this section.

Could it have its own?

Maybe a 5.4 "Authorization Grant" Or "Access Privileges"?

Reply:
You're right.. it seemed to be somewhat (but not really) okay with the original section heading "Web Application Authorization" but that didn't seem right re origin stuff, so I changed it recently to "Web Origin Authorization", but now that threw off the WAC-Allow bits.

Let's go with "5.4 Access Privilege Discovery" for now.

Reply:
https://github.com/solid/web-access-control-spec/pull/83/commits/dd2c28faea1daf575cee63c7175e3621f7f68be7